packages/edit-site/src/components/dataviews/item-actions.js

```diff
@@ -2,28 +2,18 @@
  * WordPress dependencies
  */
 import {
+	DropdownMenu,
+	MenuGroup,
+	MenuItem,
 	Button,
 	Modal,
 	__experimentalHStack as HStack,
-	privateApis as componentsPrivateApis,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { useMemo, useState } from '@wordpress/element';
 import { moreVertical } from '@wordpress/icons';
 
-/**
- * Internal dependencies
- */
-import { unlock } from '../../lock-unlock';
-
-const {
-	DropdownMenuV2Ariakit: DropdownMenu,
```

Comment:
We shouldn't revert the recent changes here. Maybe a bad rebase? By seeing this it makes me wonder if more changes in other files could slip in that shouldn't? 🤔

Reply:
It was a rebase issue, it is fixed. I did a check in other files and did not notice other cases.